LoopKit/Service/VersionCheckService.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  VersionCheckService.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 9/8/21.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+// Note: order is important for VersionUpdate.  Later version updates are more critical than earlier ones.  Do not reorder!
```

Comment:
[nit] Why is order important here?

Reply:
Because the VersionCheckServiceManager considers any service with more critical update responses to trump services with lower criticality responses